pom.xml

```
@@ -52,7 +52,7 @@
         <!-- Docker images used by both surefire and failsafe plugin -->
         <postgresql.latest.image>docker.io/library/postgres:16</postgresql.latest.image>
         <mysql.80.image>registry.access.redhat.com/rhscl/mysql-80-rhel7:latest</mysql.80.image>
-        <rhbk.image>registry.redhat.io/rhbk/keycloak-rhel9:24</rhbk.image>
+        <rhbk.image>registry.redhat.io/rhbk/keycloak-rhel9:26</rhbk.image>
```

```suggestion
        <rhbk.image>registry.redhat.io/rhbk/keycloak-rhel9:26.0</rhbk.image>
```
`podman pull registry.redhat.io/rhbk/keycloak-rhel9:26` gives me `Error: initializing source docker://registry.redhat.io/rhbk/keycloak-rhel9:26: reading manifest 26 in registry.redhat.io/rhbk/keycloak-rhel9: manifest unknown`

pulling `registry.redhat.io/rhbk/keycloak-rhel9:26.0` works fine for me

---

https://catalog.redhat.com/software/containers/rhbk/keycloak-rhel9/64f0add883a29ec473d40906?image=674f09b15573280ccffd2bb0 has `26.0 stream` label

---

Thanks for the catch, I missed that they changed version naming from `X` to `X.Y`